config/main.py

```diff
@@ -663,6 +663,17 @@ def _get_disabled_services_list(config_db):
 
     return disabled_services_list
 
+def _stop_all_services():
+    # stop all other services
+    for s in _get_sonic_services():
+        try:
+            clicommon.run_command("sudo systemctl stop {}".format(s))
```

Comment:
Instead of catching "Job Cancellation", we are also handling "Job failure" here.

Since, we're just dealing with Job Cancellation here, How about localizing the exception catch mechanism to just the "Cancellation".